core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -67,7 +67,9 @@ class EventLoggingListenerSuite extends SparkFunSuite with LocalSparkContext wit
 
   test("Basic event logging with compression") {
     CompressionCodec.ALL_COMPRESSION_CODECS.foreach { codec =>
-      testEventLogging(compressionCodec = Some(CompressionCodec.getShortName(codec)))
+      val codecShort = CompressionCodec.getShortName(codec)
+      testEventLogging(compressionCodec = Some(codecShort))
+      testEventLogging(compressionCodec = Some(codecShort.toUpperCase(Locale.ROOT)))
```

Comment:
In this test, `ALL_COMPRESSION_CODECS` corresponds to the class names, such as `org.apache.spark.io.LZ4CompressionCodec`.

If all are taken in lowercase in `CompressionCodec#getShortName`, the `if (shortCompressionCodecNames.contains(codecName))` branch should always be false. 

Moreover, even without modifications to `CompressionCodec.scala`, the newly added test cases seem to be able to pass."

Reply:
Ooh, I intended to get it checked by EventLogFileWriter.compressionCodecName. Now I realize it's not feasible as it uses fully qualified class names there.